src/mono/wasm/runtime/debug.ts

```diff
@@ -363,27 +364,31 @@ export function mono_wasm_debugger_log(level: number, message_ptr: CharPtr): voi
         return;
     }
 
-    console.debug(`Debugger.Debug: ${message}`);
+    if (Configuration === "Debug") {
```

Comment:
We want to use `INTERNAL.logging.debugger` in all configurations, but `console` only in debug configuration, correct?

Reply:
`INTERNAL.logging` is opt-in
The other thing is doing too much noise for NodeJS every time